webui/react/src/pages/WorkspaceList.tsx

```diff
@@ -322,12 +324,24 @@ const WorkspaceList: React.FC = () => {
       <Spinner spinning={isLoading}>
         {workspaces.length !== 0 ? (
           workspacesList
-        ) : (
-          <Message
-            title="No workspaces matching the current filters"
-            type={MessageType.Empty}
-          />
-        )}
+        ) :
+          (workspaceFilter === WorkspaceFilters.All && settings.archived && !isLoading) ?
```

Comment:
if we generally expect people to have the archived toggle off (it is off by default) then this wouldn't really show up.

i guess to actually know would require an additional API call or some additional data being passed in the API though